### DISCUSSION TITLE
Support grouping/folders

### DISCUSSION THREAD

**Discussion by rnsc** [xibbon/LaTerminal#3] [Ideas]
I have a bunch of hosts in different environments that could benefit from being grouped into different folders.
It's especially useful when you have more than 20 hosts, scrolling through a list to find the correct host for the correct env is a bit messy.

The application could also detect that a host is in a folder and not warn about the same alias being used by a host in the root or a different folder.
However, I'm not sure of the implications of even allowing the same alias for multiple hosts when it comes to displaying them in the recent hosts list or the Terminals list.

**Comment by rnsc** [comment]
Given that @migueldeicaza created #7 this is redundant, sorry for the noise...

**Comment by migueldeicaza** [comment]
This discussion had more context than the issue, I like that we can discuss bits here.

I wonder if it would be better to have some sort of folder structure for these, or use tags.   Tags sounds more powerful, and I could probably list the tag names before any hosts that are untagged.   Selecting that would then display the hosts that have that tag

**Reply by rnsc** [reply]
I like the idea of tagging with filtering, it would make it more elegant than just folders and probably easier to use if the tag filter is easily accessible through the list of hosts screen.

The groups/folders idea requires navigating through different views which I don't really like personally. But I don't know what solution is the simplest to implement.

**Comment by migueldeicaza** [comment]
It took longer than I wanted, it should be in a release in the next couple of weeks!

**Reply by rnsc** [reply]
Thank you @migueldeicaza !
